### ISSUE TITLE
Packer not able to login when ssh is available

### ISSUE THREAD

**Issue by clincha** [clincha/clinch-home#7]
Packer gets stuck waiting for SSH to become available with Proxmox builder and rhel8 iso. 

I'm able to ssh into the server on another client so I know it's available. 

```
C:/WINDOWS/System32/WindowsPowerShell/v1.0/powershell.exe -File build.ps1
rhel8-template-instance.proxmox.rhel8-template: output will be in this color.

==> rhel8-template-instance.proxmox.rhel8-template: Creating VM
==> rhel8-template-instance.proxmox.rhel8-template: No VM ID given, getting next free from Proxmox
==> rhel8-template-instance.proxmox.rhel8-template: Starting VM
==> rhel8-template-instance.proxmox.rhel8-template: Starting HTTP server on port 8802
==> rhel8-template-instance.proxmox.rhel8-template: Waiting 3s for boot
==> rhel8-template-instance.proxmox.rhel8-template: Typing the boot command
==> rhel8-template-instance.proxmox.rhel8-template: Waiting for SSH to become available...
```

**Comment by clincha** [comment]
https://www.packer.io/plugins/builders/proxmox/iso#qemu_agent

In the documentation, in the optional section there is this option:

**qemu_agent** - Enables QEMU Agent option for this VM. When enabled, then qemu-guest-agent must be installed on the guest. When disabled, then ssh_host should be used. Defaults to true.

So I either need to install the agent or I need to add ssh_host

**Comment by clincha** [comment]
fixed #e6b7d81893d38e0af7ca4217fa094c0de3991674

**Comment by clincha** [comment]
I installed the qemu-guest-agent in my kickstarter file and set the qumu_agent = true in the builder config.
